overview.md

```diff
@@ -0,0 +1,61 @@
+# Overview
+
+## The Endgame Plan
+
+The Endgame Plan is a proposal to overhaul and improve the ecosystem, governance, and tokenomics of the Maker Protocol. Its primary aim is for the protocol to reach a self-sustainable equilibrium called the Endgame State. In this state, the protocol is intrinsically resilient, and the scope and complexity of the core protocol will no longer change.
+
+The Endgame Plan was [introduced](https://forum.makerdao.com/t/decentralized-voter-committee-on-tuesday-5pm-cest-all-mkr-holders-welcome/15166) by Rune Christensen, the founder of Maker. After a series of initial discussions, the [MIP set](https://vote.makerdao.com/polling/QmTmS5Nf) ratifying this plan was accepted by Maker Governance on 24 October 2022. 
+
+The latest version describing the Endgame Plan is available [here](https://forum.makerdao.com/t/endgame-plan-v3-complete-overview/17427). 
+
+
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+## Goals
+
+The Endgame Plan has multiple goals which come together to create the Endgame State. A non-exhaustive list of these goals is presented below.
+- Ensure that Maker Core is more resilient to external risks such as regulatory action.
+- Ensure that the protocol can scale sustainably.
+- Realign incentives for various parties in the Maker ecosystem.
+- Reduce governance overhead for MKR holders and Maker delegates.
+- Eliminate scope creep in the core protocol.
+- Reduce MKR concentration through revised tokenomics and MKR emissions.
+
+## Core Elements
+
+The Endgame Plan consists of the following main components:
+- [SubDAOs](link.md)
+    - The Endgame Plan is based on reorganizing the existing decentralized workforce ecosystem into new self-sustainable DAOs called SubDAOs. Since each SubDAO is a DAO, it will have its own governance token, governance processes, workforce, interfaces, and users.
+    - After their launch, SubDAOs are expected to maintain a synergistic relationship with Maker Core through appropriate alignment of incentive structures.
+    - Moving much of the developmental, operational, and governance work to SubDAOs makes Maker Core simpler and more resilient.
+- [Scopes and Frameworks](scopes-and-frameworks/scopes-and-frameworks.md)
+    - Maker Core will consist of twelve Scopes. A Scope is a broad focus area for any work done to operate or improve the protocol. 
+    - The objective of introducing Scopes is to limit scope creep and ensure that any work done is aligned with Maker Core's goals.
+    - Frameworks define the rules that SubDAOs must adhere to. Frameworks are expected to be refined over time to maximize profit and minimize risk to Maker Core.
```

Comment:
I think the term for these is still 'Scope Frameworks' rather than just frameworks, maybe Payton can confirm.

Reply:
Yeah, it gets a bit confusing. But Scope refers to the problem area and Scope Framework is the specific rules for it. So I think here saying "Scope Frameworks" is most accurate, I would only drop Scope from the phrase when it's implied like under a specific scope section or something